pkg/pipelines/init.go

```diff
@@ -114,6 +114,12 @@ func Init(o *InitParameters, fs afero.Fs) error {
 		return err
 	}
 	_, err = yaml.WriteResources(fs, o.OutputPath, outputs)
+
+	orgRepo, _ := orgRepoFromURL(o.GitOpsRepoURL)
+
+	if err == nil {
+		log.Successf(fmt.Sprintf("Intialised the %s repo at %s!!", orgRepo, o.OutputPath))
+	}
```

Comment:
o.OutputPath can be empty, right?

Reply:
Can we just say "Intialised the %s repo at %s!!", orgRepo" leaving the output path or do we calculate the output path rather than the o.OutputPath?

Reply:
Leaving the outputpath out is fine.    The intend of a brief message is to provide feedback to the user that command has done something with no errors.   We want to be brief.